src/component/redux/put_to_do.js

```diff
@@ -1,33 +1,19 @@
 import React from "react";
-import PropTypes from "prop-types";
 import { AddButton } from "../../style/css_element";
 import { useDispatch, useSelector } from "react-redux";
-function Put_To_Do() {
-  const text = useSelector((state) => state.input);
-  const id = useSelector((state) => state.id);
+import { ActionType } from "../../Action/ActionType";
+function Put_To_Do({ text, setText }) {
```

Comment:
Not a blocker, but just wonder why remove `PropTypes` here?

Reply:
when using redux it didn't any props need to define type , so remove it .But after first time review , props (text,setText) have been added back.And I forgot to use proptypes define the type.XD....